scripts/build-plain-pipelines.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Then script file name shouldn't have "build-" in it. (The others have it only because they say "build app" or "build bosh release"